website_sale_template_adj/views/website_sale_template_adj.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template
+        id="website_sale_templates_view_adj"
+        inherit_id="website_sale.payment_sale_note"
+    >
+        <xpath expr="//label[hasclass('custom-control-label')]" position="after">
+            <span>
+                <br />
+                according to the current agreement.
+            </span>
+        </xpath>
```

Comment:
<xpath expr="//label[hasclass('custom-control-label')]" position="attributes">
            <attribute name="invisible">1</attribute>
        </xpath>
        <xpath expr="//input[hasclass('custom-control-input')]" position="after">
            <label for="checkbox_cgv" class="custom-control-label">
                We agree to the <a target="_BLANK" href="/shop/terms">terms &amp; conditions</a>
            </label>
            <span>
                <br />
                according to the current agreement.
            </span>
        </xpath>

Reply:
I suggest this. "I -> We"

Reply:
- We tried also bellow, but it doesn't work as expected.
- Once, I'll update with minimal adjustments.

       <xpath expr="//div[hasclass('custom-control')]" position="attributes">
            <attribute name="class">hidden</attribute>
        </xpath>
        <xpath expr="//input[hasclass('custom-control-input')]" position="after">
            <label for="checkbox_cgv" class="custom-control-label">
                 We agree to the <a target="_BLANK" href="/shop/terms">terms & conditions</a>
            </label>
                <span>
                <br />
                    according to the current agreement.
                </span>
        </xpath>